lib/schemaUtils.js

```diff
@@ -1832,11 +1832,21 @@ module.exports = {
           }
         }
 
-        param = new sdk.FormParam({
-          key: key,
-          value: value,
-          type: 'text'
-        });
+        // Set param type to 'file' for binary value
+        if (value === '<binary>') {
```

Comment:
@webholik This will not work when value is set to `example` instead `schema`, in those cases value will be faked string or example value. Can you find another way instead relying on value?